Observer/ImportRates.php

```diff
@@ -339,17 +324,19 @@ private function _importRates()
      *
      * @return OperationInterface
      */
-    private function makeOperation($rates, $bulkUuid): OperationInterface {
+    private function makeOperation($rates, $bulkUuid, $topic): OperationInterface
```

Comment:
Let's abstract this method out a little further so that we aren't unnecessarily including the shipping_class, product_tax_classes, and customer tax classes parameters when making the delete rate operations. Perhaps a makeDeleteRatesOperation and makeCreateRatesOperation that calls a more generic make rates operation?